fibo/src/styles/setup/_bootstrap-override.scss

```diff
@@ -1,5 +1,5 @@
 body {
-  font-family: "Roboto", sans-serif;
+  font-family: "Inter", sans-serif;
```

Comment:
Is there any reason to change the font in the project to this one?

Reply:
> Is there any reason to change the font in the project to this one?

It's the font used in figma desgin and it is used in ontology view already but not in other pages, this is update to the global style. Currently you can see the difference in production:
https://spec.edmcouncil.org/fibo/ontology (Inter)
https://spec.edmcouncil.org/fibo/ (Roboto - old font)